hooks/configure

```diff
@@ -74,6 +74,24 @@ switch_service() {
 	esac
 }
 
+update_pi_config_value() {
+    path="$1"
+    k="$2"
+    v="$3"
+
+    if [ -z "$v" ]; then
+        # empty - unset an option
+        sed -i -e "/^[ \t#]*${k}=/ s/^#*/#/" "$path"
+    else
+        # non-empty - set an option
+        if grep -q "^[ \t#]*${k}=" "$path"; then
+            sed -i -e "s/^[ \t#]*${k}=.*/$k=$v/" "$path"
+        else
+            echo "$k=$v" >> "$path"
+        fi
```

Comment:
Woah, that is serious sed! I update the code (slightly differently in a multi-line sed script, found it slightly easier to read this way, but maybe just me).